provider/maas/maas2_environ_whitebox_test.go

```diff
@@ -1543,11 +1543,11 @@ func (suite *maas2EnvironSuite) TestAllocateContainerAddressesMachinesError(c *g
 	c.Assert(err, gc.ErrorMatches, "boom")
 }
 
-func getArgs(c *gc.C, calls []testing.StubCall) interface{} {
-	c.Assert(calls, gc.HasLen, 1)
-	args := calls[0].Args
-	c.Assert(args, gc.HasLen, 1)
-	return args[0]
+func getArgs(c *gc.C, calls []testing.StubCall, callNum, argNum int) interface{} {
+	c.Assert(len(calls) >= callNum, jc.IsTrue)
```

Comment:
If you use `c.Assert(len(calls), gc.Not(jc.LessThan), callNum)` here then if it fails it'll show us how many calls there were.